mayastor/src/bdev/nexus/mod.rs

```diff
@@ -60,16 +60,14 @@ pub fn nexus_instance_new(name: String, size: u64, children: Vec<String>) {
     list.push(Nexus::new(&name, size, None, Some(&children)));
 }
 
-/// called during shutdown so that the bdevs for each child is destroyed first
-pub async fn nexus_destroy_all() {
-    info!("destroying all nexuses...");
+/// called during shutdown so that all nexus children are in Destroying state
+/// so that a possible remove event from SPDK also results in bdev removal
+pub async fn nexus_children_to_destroying_state() {
+    info!("setting all nexus children to destroying state...");
     for nexus in instances() {
-        if let Err(e) = nexus.destroy().await {
```

Comment:
Can you please clarify where the nexus is destroyed, since nexus.destroy() is removed by this fix ?

Reply:
It's not explicitly destroyed, this essentially restores the original behaviour prior to #661. The nexus instances themselves are cleared as part of shutdown when `NexusModule::nexus_mod_fini()` is called.

Reply:
Thanks for clarification.